### PR TITLE
Cleanup ViewListener

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

**Comment by ADmad** [comment]
Are `config/form.php` and `config/paginator.php` actually used?  I think not. They shouldn't be required given the use of BUI.

**Comment by jadb** [comment]
:+1:

File: config/defaults.php [hunk 1]
```diff
@@ -4,15 +4,15 @@
     'CrudView' => [
         'css' => [
             'https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.4/css/bootstrap.css',
-            'CrudView./eonasdan-bootstrap-datetimepicker/css/bootstrap-datetimepicker.min.css',
+            'https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datetimepicker/4.14.30/css/bootstrap-datetimepicker.min.css',
```
Comment (jippi) [inline config/defaults.php]: the downside to using the CDN for Bownty specifically is that we use vagrant, and can't always depend on internet - it's there 99% of the time though, so not a huge deal breaker

Comment (ADmad) [inline config/defaults.php]: I did so because there were other assets already being included using CDN.

Comment (ADmad) [inline config/defaults.php]: Plus you can override the default config if needed :)

**Pull request merged**